api/tests/unit/domain/models/User_test.js

```diff
@@ -3,7 +3,7 @@ const config = require('../../../../lib/config');
 const localeService = require('../../../../lib/domain/services/locale-service');
 const User = require('../../../../lib/domain/models/User');
 
-describe('Unit | Domain | Models | User', function () {
+describe.only('Unit | Domain | Models | User', function () {
```

Comment:
```suggestion
describe('Unit | Domain | Models | User', function () {
```

C'est bizarre que tu arrives à push le code avec un `.only`. Tu devrais activer husky 🐶

Reply:
Oups 🙈